### ISSUE TITLE
No visual /keyboard focus is present on dropdown elements except the element already in selected state while navigating in "Package source" dropdown in High contrast black mode.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#386]
### Steps to Reproduce

1. Change the High contrast standards to High Contrast black before opening the application.
2. Launch Xamarin Workbooks application and create a new workbook.
3. Navigate to File -> Add Package. Invoke "Add Package" . It will open "Package Manager" window.
4.Navigate to "Package source" dropdown and invoke it by "Alt+down arrow". Navigate through the Package source dropdown.

### Expected Behavior
Dropdown elements should get visual focus while navigating in the dropdown
### Actual Behavior
Except the element in selected state, rest of the elements in the "Package source" dropdown of package manager window are not getting the visual keyboard focus around them in high contrast black mode.

Note: In high contrast white mode dotted rectangular focus appears on Package source dropdown elements.
### User Impact
As there is no indication of focus, user will not know whether the element in the dropdown has got focus or not.
### Screenshots
![23 hcb](https://user-images.githubusercontent.com/22956130/40764015-e2fb249e-64c4-11e8-9c30-68064d0112bf.png)
![23 hcw](https://user-images.githubusercontent.com/22956130/40764016-e32b782e-64c4-11e8-8556-59708804bc1c.png)
![23 normal mode](https://user-images.githubusercontent.com/22956130/40764017-e359a370-64c4-11e8-8250-bb6293c9d2e6.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS4.3.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
